major mods for cmdline params, one script for desktop/android, etc.  …

jesup opened the pull request:
…run_test.py is now the main test

jesup committed:
major mods for cmdline params, one script for desktop/android, etc.  run_test.py is now the main test

jesup committed:
major mods for cmdline params, one script for desktop/android, etc.  run_test.py is now the main test

acreskeyMoz commented on run_test.py:
```diff
@@ -0,0 +1,259 @@
+    "--iterations",
+    type=int,
+    default=3,
+    help="Number of iterations",
```
Nice, these arguments are a good addition.

acreskeyMoz commented:
@jesup 
There is a lot of good stuff in here.
Let me take a stab at integrating it because I still want to keep this as minimal as possible, while adding things like the options and the desktop/android switching.

I've pushed this PR to a branch in case you need it directly in the future.
https://github.com/acreskeyMoz/browsertime_on_android_scripts/tree/jesup_commandline

acreskeyMoz committed:
Removed cold startup note from readme.md

acreskeyMoz committed:
Refactored to use arguments + progress bar

Also updated the site list

jesup committed:
add prefs, fix some bugs, support for local wpr

jesup committed:
add prefs, fix some bugs, support for local wpr

jesup committed:
Add comparative prefs, other things

jesup committed:
Add comparative prefs, other things

acreskeyMoz committed:
Added variant file definition and features from jesup

Docs updated to reflect android/desktop capabilities

acreskeyMoz committed:
Cleaned up readme.me

acreskeyMoz committed:
Removed extranneous browsertime bin path

acreskeyMoz committed:
Removed browsertime output

acreskeyMoz committed:
Merge branch 'main' of github.com:acreskeyMoz/browsertime_on_android_scripts into main

acreskeyMoz committed:
Removed pycache

acreskeyMoz committed:
Update README.md

acreskeyMoz committed:
Added options to specify paths + small improvements

Added these options:
-specify output path (defaults to "browsertime-results")
-specify geckodriver path
-specify browsertime path
-precache cache shaders when WebRender is specified to avoid first-compile hit
-option to restart adb between variants (workaround to adb file descriptor leak)

acreskeyMoz committed:
Updated readme.md from new options

acreskeyMoz committed:
Update variants_android.py

typo

acreskeyMoz committed:
Fix to output path

acreskeyMoz committed:
Enabled webdriver pageload by default

acreskeyMoz committed:
Merge branch 'main' of github.com:acreskeyMoz/browsertime_scripts into main

acreskeyMoz committed:
Defaulted keep original video to false to reduce storage use

acreskeyMoz committed:
Update sites.txt

jesup committed:
Add a bunch of commands to run_test.py; added mitm support

jesup committed:
Add a bunch of commands to run_test.py; added mitm support

jesup committed:
mitm comments

jesup committed:
resolve merge issues

jesup committed:
resolve pull merge issues

jesup commented:
FYI, I've updated this with all my changes (mitm support, etc).   It's merged with your current main, and base tests work.  lots of irrelevant stuff is in there; focus on run_test.py, desktop.sh, reload.sh, variants_fission.py, perf.sh
